google/cloud/spanner/admin/integration_tests/instance_admin_integration_test.cc

```diff
@@ -214,6 +214,44 @@ TEST_F(InstanceAdminClientTest, InstanceCRUDOperations) {
   EXPECT_STATUS_OK(client_.DeleteInstance(in.FullName()));
 }
 
+TEST_F(InstanceAdminClientTest, CreateInstanceStartAwait) {
+  if (!Emulator() && !RunSlowInstanceTests()) {
+    GTEST_SKIP() << "skipping slow instance tests; set "
+                 << "GOOGLE_CLOUD_CPP_SPANNER_SLOW_INTEGRATION_TESTS=instance"
+                 << " to override";
+  }
+
+  std::string instance_id = spanner_testing::RandomInstanceName(generator_);
+  Instance in(ProjectId(), instance_id);
+  ASSERT_FALSE(in.project_id().empty());
+  ASSERT_FALSE(in.instance_id().empty());
```

Comment:
nit: remove?

I see that the above test does the same thing, but `RandomInstanceName` is never going to return an empty instance the way `ProjectId()` or `InstanceId()` might.